### ISSUE TITLE
Save Routine for New Recipes fails on Clicking Save Button

### ISSUE THREAD

**Issue by Oldbwl** [kirienko/gourmet#196]
**Actions**
1. Run Gourmet
2. Click "New Recipe
3. Populate any or all fields (include title)
4. Click "Save"

**Results**
1. Save Button turns Grey whilst save is taking place. (New Recipe card remains open as expected)
2. No change noted to underlying Recipe index - Clicking "Open Recipe card" opens a semi blank screen - the image is there but the text is missing
3. Closing New Recipe Card elicits Warning Notice that recipe has not been saved  - Clicking Save here results in Recipe creation and underlying Index screen visibly updating

**Expected Results**
1. Clicking Save within New Recipe Card should save results at that point

**Comment by Oldbwl** [comment]
Further noted it is necessary to close application and be presented with another - You recipe has not saved message  and reopen application before the recipe updates and displays correctly

**Comment by cydanil** [comment]
Out of curiosity, did you happen to make a typo when editing this new recipe?

**Comment by Oldbwl** [comment]
> Out of curiosity, did you happen to make a typo when editing this new recipe?

I do make typos - and some exist from the original authors who's work I am using as source. But I am not aware that I made any spelling changes.  - indeed none were flagged as errors - I presumed spell checking was disabled. The workflow was consistent across 2 machines and different recipes, with the the same result. - I just tried to miss-spell, and got no red flag. Also, was only allowed to make one save at a time. Not all my subsequent changes were spotted prompting a save.
